plugins/keywords-generator/src/main/kotlin/org/jetbrains/dataframe/keywords/KeywordsGeneratorPlugin.kt

```diff
@@ -0,0 +1,44 @@
+package org.jetbrains.dataframe.keywords
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.tasks.SourceSet
+import org.gradle.api.tasks.SourceSetContainer
+import org.gradle.kotlin.dsl.get
+import org.gradle.kotlin.dsl.register
+import org.jetbrains.kotlinx.dataframe.BuildConfig
+import java.io.File
+
+abstract class KeywordsGeneratorPlugin : Plugin<Project> {
+
+    override fun apply(target: Project): Unit = with(target) {
+        // from https://kotlinlang.org/docs/whatsnew21.html#compiler-symbols-hidden-from-the-kotlin-gradle-plugin-api
+        val taskDependencyScope = configurations.create("keywordsGeneratorScope")
```

Comment:
For me "scope" here was misleading, took me longer to realize it simply is a new configuration

Reply:
just took it from https://kotlinlang.org/docs/whatsnew21.html#compiler-symbols-hidden-from-the-kotlin-gradle-plugin-api

Reply:
It represents a `DependencyScopeConfiguration`, will fail in gradle 9.0 if we don't split it